### PR TITLE
Update to PHPUnit 7

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: tests/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/Value/UniqueValueDenormalizerTest.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function testReturnsUniqueValueIfUniqueFlagsFound()
         $result = $denormalizer->denormalize($fixture, $flags, $value);
 
         $this->assertInstanceOf(UniqueValue::class, $result);
-        $this->stringContains('dummy_id', $result->getId());
+        $this->assertStringStartsWith('Dummy#', $result->getId());
```
Comment (carusogabriel) [inline tests/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/Value/UniqueValueDenormalizerTest.php]: Was `stringContains` use in purpose?

Comment (theofidry) [reply tests/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/Value/UniqueValueDenormalizerTest.php]: I'm fine with changing this if this is a issue I don't think there was a strong reasons for it.

**Comment by theofidry** [comment]
👍 tests seems to be failing with lowest deps though. It looks like we might need to dump the minimal requirements of some deps (likely phpunit bridge)

**Comment by carusogabriel** [comment]
@theofidry Sure, gonna look into that and update this PR ASAP :)

**Comment by theofidry** [comment]
Great, thanks :)

**Pull request merged**